src/StackExchange.Redis/KeyspaceIsolation/DatabaseWrapper.cs

```diff
@@ -576,6 +576,14 @@ public RedisValue[] SortedSetRangeByRank(RedisKey key, long start = 0, long stop
             return Inner.SortedSetRangeByRank(ToInner(key), start, stop, order, flags);
         }
 
+        public long SortedSetRangeAndStore(RedisKey destinationKey, RedisKey sourceKey, RedisValue start,
+            RedisValue stop,
+            SortedSetOrder sortedSetOrder = SortedSetOrder.ByRank, Exclude exclude = Exclude.None,
+            Order order = Order.Ascending, long skip = 0, long take = 0, CommandFlags flags = CommandFlags.None)
```

Comment:
Global suggestions here: reverse source/dest (more in line with most APIs/.NET-isms), let's go clean with the line breaks, and make `take` default to `null` instead of `0` since that's more accurate for "unlimited" rather than using a sentinel value. Skip does makes sense as "skipping 0", though.
```suggestion
        public long SortedSetRangeAndStore(
        	RedisKey sourceKey, 
        	RedisKey destinationKey, 
        	RedisValue start, 
        	RedisValue stop,
            SortedSetOrder sortedSetOrder = SortedSetOrder.ByRank, 
            Exclude exclude = Exclude.None,
            Order order = Order.Ascending, 
            long skip = 0, 
            long? take = null, 
            CommandFlags flags = CommandFlags.None)
```

Reply:
dest->source is the order of the parameters in the command, that said I've made the mistake more than once of ordering it source->dest so as long as you're ok with that inconsistency, sure thing 😂